app/serializers/base_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class BaseSerializer
+  include ActiveModel::Serialization
+
+  def to_hash
+    self.serializable_hash.compact
+  end
+
+  def to_json
```

Comment:
If you add the (*args) here we could avoid calling `to_json` on render.

```suggestion
  def to_json(*args)
    to_hash.to_json(*args)
  end
```

It’s not necessary to call to_json on the object you want to render since render will automatically do that for you.(https://apidock.com/rails/ActionController/Base/render)

Reply:
Good point! I'll change it.

Reply:
dbee4cd7031f697c42d3f4b1ece3f48019fae5ab